WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsViewController.swift

```diff
@@ -777,11 +777,17 @@ extension NotificationsViewController {
         if let postID = note.metaPostID,
             let siteID = note.metaSiteID,
             note.kind == .matcher || note.kind == .newPost {
-            let readerViewController = ReaderDetailViewController.controllerWithPostID(postID, siteID: siteID)
-            readerViewController.navigationItem.largeTitleDisplayMode = .never
-            readerViewController.hidesBottomBarWhenPushed = true
-            readerViewController.coordinator?.notificationID = note.notificationId
-            displayViewController(readerViewController)
+
+            if isSidebarModeEnabled && splitViewController == nil {
```

Comment:
Yes, but after a quick search I couldn't find a reliable way to check if something is displayed in a popover. I guess the `modalPresentationStyle` check could work in this case. I'm a bit reluctant to use it because it's only not a completely reliable indicator as on iPhone `.popover` is displayed in a sheet by default (obviously not the case here for this screen).

Btw, I'll try to remove `isSidebarModeEnabled` as well during these couple of weeks. It was added there as a temporary measure and a type of a feature flag.